drizzlepac/hapsequencer.py

```diff
@@ -686,6 +687,14 @@ def run_hap_processing(input_filename, diagnostic_mode=False, input_custom_pars_
         logging.exception("message")
 
     finally:
+        # Try to ensure there is a manifest filename if the code exits in the poller_utils.py
+        # so the shutdown is tidy
+        ntokens = len(input_filename.split("_"))
+        if manifest_name == "" and input_filename.lower().endswith("input.out") and ntokens == 4:
```

Comment:
It doesn't need to be done as part of this PR, but this does add another dependency in the code on the input file ending with 'input.out'.  This is not a 'bad' think, but simply something that should be clearly documented somewhere for this code.

Reply:
I understand. The pipeline files are currently defined in this manner which is what this PR is addressing, and there is the fallback for a generic naming convention which could be supplied by a user for their personal processing.